debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/SnapshotProcessor.java

```diff
@@ -155,6 +155,7 @@ private void takeTableSnapshot(TableMetadata tableMetadata) throws IOException {
         LOGGER.info("Executing snapshot query '{}' with consistency level {}", statement.getQueryString(), statement.getConsistencyLevel());
         ResultSet resultSet = cassandraClient.execute(statement);
         processResultSet(tableMetadata, resultSet);
+        LOGGER.debug("The snapshot of table '{}' has been taken", tableName(tableMetadata));
```

Comment:
Related to that, it'd also be nice to look into metrics. We'd get those "for free", when moving the Cassandra connector to the framework used by the other Debezium connectors :)